src/RacingGame.js

```diff
@@ -0,0 +1,70 @@
+import { inputHandler, outputHandler } from './views/index.js';
+import { GAME_MESSAGES, RULES } from './constants/index.js';
+import {
+  carNameParser,
+  randomNumber,
+  validateCarNames,
+} from './utils/index.js';
+import { Car } from './models/index.js';
+
+class RacingGame {
+  #cars;
+  #tryCount;
+
+  async startGame() {
+    await this.#initialize();
+    await this.#play();
+    await this.#finish();
```

Comment:
```suggestion
    this.#play();
    this.#finish();
```
`play`, `finish` 메서드에는 비동기 로직이 없는 것 같은데 `await` 키워드를 사용하신 이유가 있으신가요??

Reply:
앗 그렇네요 리팩토링 하면서 삭제를 안했던 것 같아요!! 완전 꼼꼼하신... 👍